src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -9,19 +9,19 @@
  */
 public class DriverManager {
 
-	private Job2dDriver currentDriver = new LoggerDriver();
+    private Job2dDriver currentDriver = new LoggerDriver();
 
-	/**
-	 * @param driver Set the driver as current.
-	 */
-	public synchronized void setCurrentDriver(Job2dDriver driver) {
-		currentDriver = driver;
-	}
+    /**
+     * @return Current driver.
+     */
+    public synchronized Job2dDriver getCurrentDriver() {
+        return currentDriver;
+    }
 
-	/**
-	 * @return Current driver.
-	 */
-	public synchronized Job2dDriver getCurrentDriver() {
-		return currentDriver;
-	}
+    /**
+     * @param driver Set the driver as current.
+     */
+    public synchronized void setCurrentDriver(Job2dDriver driver) {
+        currentDriver = driver;
+    }
```

Comment:
Ok, I understood the problem, in other hand I think, that including formatting provider in xml file (that can be imported into IDE and used by every developer working on this project) should be good solution to avoid issues like this above.